test/integration/targets/unarchive/tasks/test_different_languave_var.yml

```diff
@@ -0,0 +1,48 @@
+- name: test non-ascii with different LANGUAGE
```

Comment:
This file needs to be called from tasks/main.yml.

Minor nit: filename has a typo